src/unix/process.c

```diff
@@ -34,8 +34,12 @@
 #include <poll.h>
 
 #if defined(__APPLE__) && !TARGET_OS_IPHONE
+#include <spawn.h>
+#include <sys/kauth.h>
+#include <dlfcn.h>
 # include <crt_externs.h>
 # define environ (*_NSGetEnviron())
+#include "darwin-stub.h"
```

Comment:
indentation fix:
```suggestion
# include <sys/kauth.h>
# include <dlfcn.h>
# include <crt_externs.h>
# define environ (*_NSGetEnviron())
# include "darwin-stub.h"
```
why's spawn.h listed in both files? actually, seems possible it's only needed here?